src/main/java/com/softserve/kickscooter/kickscootervehicle/management/dao/Scooter.java

```diff
@@ -0,0 +1,41 @@
+package com.softserve.kickscooter.kickscootervehicle.management.dao;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "scooter")
+public class Scooter {
```

Comment:
JPA entities should be JavaBean's i.e. have equals and hashcode. What's the problem in using `@Data` annotation?